src/python/pants/base/exiter.py

```diff
@@ -72,8 +57,13 @@ def exit(self, result=0, msg=None, out=None):
     """
     if msg:
       if PY2:
-        msg = msg.encode('utf-8')  # sys.stderr expects bytes in Py2, unicode in Py3
-      print(msg, file=out or sys.stderr)
+        # sys.stderr expects bytes in Py2, unicode in Py3
```

Comment:
Should use `ensure_bytes` then, probably. See Eric's comment on #6593

Reply:
If you're referring to `pants.util.strutil.ensure_binary`, we now use it here.